shell/edit/logging-flow/index.vue

```diff
@@ -51,15 +57,17 @@ export default {
   mixins: [CreateEditView],
 
   async fetch() {
-    const hasAccessToClusterOutputs = this.$store.getters[`cluster/schemaFor`](LOGGING.CLUSTER_OUTPUT);
-    const hasAccessToOutputs = this.$store.getters[`cluster/schemaFor`](LOGGING.OUTPUT);
+    const currentCluster = this.$store.getters['currentCluster'];
+    const inStore = currentCluster.isHarvester ? VIRTUAL : 'cluster';
```

Comment:
Could this be switched to use `this.$store.getters['currentProduct'].inStore`?

Reply:
I think here we can switch to `this.$store.getters['currentProduct'].inStore`